open_academy/views/view_course.xml

```diff
@@ -0,0 +1,9 @@
+<odoo>
+    <record id="action_menu_courses" model="ir.actions.act_window">
```

Comment:
Follow this [documentation](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#252security-view-and-action)